x/nameservice/client/rest/rest.go

```diff
@@ -66,7 +67,7 @@ func buyNameHandler(cdc *codec.Codec, cliCtx context.CLIContext) http.HandlerFun
 			return
 		}
 
-		rest.CompleteAndBroadcastTxREST(w, r, cliCtx, baseReq, []sdk.Msg{msg}, cdc)
+		clientrest.CompleteAndBroadcastTxREST(w, cliCtx, baseReq, []sdk.Msg{msg}, cdc)
```

Comment:
If you make changes in the `.go` files please make the same changes in the associated `.md` file in the `/tutorial`

Reply:
Sorry, I was negligent, and commit rest.md just now.